app/models/case/sar/offender_complaint.rb

```diff
@@ -5,6 +5,43 @@ class Case::SAR::OffenderComplaint < Case::SAR::Offender
   validates_presence_of :original_case
   after_create :stamp_on_original_case
 
+  jsonb_accessor :properties,
+                 complaint_type: :string,
+                 complaint_subtype: :string,
+                 priority: :string
+
+  validates :complaint_type, presence: true
+  validates :complaint_subtype, presence: true
+  validates :priority, presence: true
+
+  enum complaint_type: {
+    standard:  'standard',
+    ico: 'ico',
+    litigation: 'litigation',
+  }
+
+  enum complaint_subtype: {
+    missing_data:  'missing_data',
+    inaccurate_data: 'inaccurate_data',
+    redacted_data: 'redacted_data',
+    timeliness: 'timeliness',
+  }
+
+  enum priority: {
+    normal:  'normal',
```

Comment:
Doesn't priority already exist? Not sure there is a need to have 'normal' at all given that is the default. Is it not just a boolean flag to say something is high priority 🤔

Reply:
Great point... 🤔 I guess I've approached it from the point of view that there are really three priorities but we're collapsing them into two. Using the enum even with a boolean automatically gives us a full set of methods we can use to query and set the value, and using the strings makes it easier if we end up extending this beyond two states one day.